google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/slow/ITBackupTest.java

```diff
@@ -717,7 +717,7 @@ private void testPagination() {
     Page<Backup> page = dbAdminClient.listBackups(instanceId, Options.pageSize(1));
     assertEquals(1, Iterables.size(page.getValues()));
     numBackups++;
-    assertTrue(page.hasNextPage());
+    assertFalse(page.hasNextPage());
```

Comment:
This is a bit surprising. Did this always fail before? Or is there some other reason that this needs to be changed now?

Reply:
No, that is not what that means. That value is for how many items per page that the backend should return. So it basically works like this:
1. If you have a query that returns 10 results, and you set page size to 1, then you get 10 pages with each 1 item in each page.
2. However, if you execute the same query with page size 5, then you get 2 pages with each 5 items.

Reply:
Yeah I meant the same. In this example, there's only one backup so it will not have next page.